packages/grafana-ui/src/components/Forms/Field.tsx

```diff
@@ -15,8 +15,6 @@ export interface FieldProps extends HTMLAttributes<HTMLDivElement> {
   children: React.ReactElement;
   /** Label for the field */
   label?: React.ReactNode;
-  /** Forcibly use a Label, despite passing a non-string node. */
-  useLabel?: boolean;
```

Comment:
Is this an intentional change for this PR?

Reply:
Yes, this reverts a change from https://github.com/grafana/grafana/pull/102687, which wasn't necessary.